### PR TITLE
Initial merge

### PR DESCRIPTION
**Pull request by ecv2115**

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+const faker = require('faker');
+const { addPhoto } = require('./controllers/controllers');
```
Comment (marketgai) [inline seed.js:2]: not sure this is needed, you arent calling it anywhere

File: seed.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+const Photo = require('./models/Photo');
+const dbConnection = require('./db/connection');
```
Comment (marketgai) [inline seed.js:4]: not sure you're using this anywhere either.

File: seed.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const restaurant_id_arr = [1, 2, 3, 4, 5]
```
Comment (marketgai) [inline seed.js:5]: probably will use a different way of determining this. you are going to need more than 5 restaurants.

File: seed.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  let record = {
+    "url": `${faker.image.food()}`,
+    "description": `${faker.lorem.sentence()}`,
+    "date": `${faker.date.past()}`,
+    "user": {
+      "username": `${faker.internet.userName()}`,
+      "friends": randomFriends,
+      "reviews": randomReviews
+    },
+    "restaurant_id": restaurant_id_arr[randomRestaurant_id]
```
Comment (marketgai) [inline seed.js:20]: will probably need to refactor

File: controllers/controllers.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
+const Photo = require('../models/Photo');
+
+module.exports.getPhotos = async (req, res, next) => {
```
Comment (Advaita1) [inline controllers/controllers.js:3]: great job using async await!

**Review by marketgai — CHANGES_REQUESTED** [review]
looks good but you probably have to refactor to add some other pieces of data and accomodate more  restaurants

**Pull request merged**